03-events-forms-refs-life-cycle/life-cycle/gifs/src/GifForm.js

```diff
@@ -0,0 +1,78 @@
+import React, {Component} from 'react'
+
+
+class GitForm extends Component{
+    
+    constructor(props){
+    	super(props);
+    	this.state={
+            category : ""
+    	}
+    	this.handleChange = this.handleChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.removeGif = this.removeGif.bind(this);
+    }
+    
+    removeGif(){
+    	return this.props.removeGifs();
```

Comment:
this method is not necessary, since you can just call `this.props.removeGifs()` anywhere you would call `this.removeGif()`